src/bank.ts

```diff
@@ -88,6 +88,21 @@ export class Bank extends ServerNode {
   // TODO: POST /validator_confirmation_services
 
   // TODO: POST /upgrade_notice
+  /**
+   * Sends a signed POST request to the bank for an upgrade notice.
+   * @param nodeIdentifier the node identifier of the bank that is receiving the upgrade notice
+   * @param bankAccount the current bank server's account
+   */
+  async sendUpgradeNotice(nodeIdentifier: string, bankAccount: Account): Promise<any> {
```

Comment:
Please remove the return type for the method, sorry that my prompt included it as it is implicit.